### ISSUE TITLE
Don't build website as part of prepack

### ISSUE THREAD

**Issue by cowboyd** [thefrontside/bigtest#259]
The `prepack` script is for preparing an NPM module for release: things like compiling typescript, any code-gen that needs to happen, etc.... As such, we do it quite a bit in the dev process, in order to consume modules from one another.

The website, which is not a consumable npm package, gets built during this phase which makes the builds take a long time, and does not serve any useful purpose.

Let's pull the website build and deploy out into a separate prepack script.

1. Can we decouple the website build into a separate process?
2. Does website need to be in the /packages at all?

Among other things, this means that a yarn install for a development will not download an asteroid belt of Gatsby dependencies.

**Comment by minkimcello** [comment]
I believe I've looked into this in the past and there's no way of leaving out a package when using `yarn workspaces` command.

I don't remember what @taras said but I've also inquired about bringing the website out from the packages directory.

I don't think we need a `prepack` script for the website package anyway. We could just modify the `prepack` script to an `echo`. Netlify uses `gatsby build` command by default and doesn't rely on the `prepack` script and nor does `gatsby develop`.
